installer/dev/main.cpp

```diff
@@ -6,6 +6,11 @@
 #include "install.h"
 #include "resource.h"
 
+#ifndef GUID_NULL
```

Comment:
I take it that some new changes in this PR called for this definition.

Reply:
yes, the MsUCRT headers/libs introduced this (and the pattern is common in OS code)